magpie-aws/src/test/java/io/openraven/magpie/plugins/aws/discovery/services/IAMAccountDiscoveryIT.java

```diff
@@ -58,17 +59,16 @@ private void assertConfiguration(MagpieEnvelope envelope) {
     var passwordPolicy = supplementaryConfiguration.get("PasswordPolicy");
     assertEquals("{\"minimumPasswordLength\":8,\"requireSymbols\":true,\"requireNumbers\":true,\"requireUppercaseCharacters\":true,\"requireLowercaseCharacters\":true,\"allowUsersToChangePassword\":true,\"expirePasswords\":true,\"maxPasswordAge\":200,\"passwordReusePrevention\":1,\"hardExpiry\":true}",
       passwordPolicy.toString());
-    assertNotNull(supplementaryConfiguration.get("summaryMap").toString());
```

Comment:
We removed accountSummary discovery here https://github.com/openraven/magpie/pull/115

Don't remember what was the reason

Reply:
It wasn't removed it was moved from supplementaryConfiguration to configuration

Reply:
Ah, I see it now. Thanks! We lost it somewhere during transition to `MagpieResource`. Pushed fix